Documentation/configuration/metrics.rst

```diff
@@ -34,6 +34,9 @@ Endpoint
 * ``endpoint_count``: Number of endpoints managed by this agent
 * ``endpoint_regenerating``: Number of endpoints currently regenerating
 * ``endpoint_regenerations``: Count of all endpoint regenerations that have completed, tagged by outcome
+* ``endpoint_regeneration_seconds``: Total time taken to regenerate endpoints
+* ``endpoint_regeneration_square_seconds``: Sum of squares of total time taken to regenerate endpoints
```

Comment:
```
... Total sum of endpoint regeneration times
... Total sum of squares of endpoint regeneration times
```

Reply:
sure.

Reply:
done. Changed granularity to milliseconds for more realistic prometheus metrics gathering